.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,20 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
 
 ---
 
+## Is your feature request related to a problem? Please describe
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
 
+## Describe the solution you'd like
```

Comment:
nit: the comment asks for the desired behaviour rather than how that behaviour is to be implemented (the solution). 

```suggestion
## Describe the behaviour you'd like
```